test/augmentation/test_augmentation.py

```diff
@@ -961,8 +961,33 @@ def test_random_vflip_coord_check(self, device, dtype):
             == input[..., input_coordinates[0, 1, :], input_coordinates[0, 0, :]]
         ).all()
 
+    @pytest.mark.skip(reason="not implemented yet")
+    @pytest.mark.nn
```

Comment:
```suggestion
```
not very clear we need this anymore, unless we find a real utility

Reply:
Thanks for noticing, i will fix it everywhere